flows/index.md

```diff
@@ -0,0 +1,128 @@
+
+Apart from the extractor(s), you'll want to add selected transformations and writers that will load your transformed data into your selected data destination.
+Continue adding other ready-made components and organize them in a logical flow,  or select those you want to run in parallel using drag-and-drop function. 
+
+- Organize the components into a logical flow using the drag-and-drop functionality.
+- Configure each component by providing the credentials and instructions for what or where to extract or write or for what code to execute in a transformation.
+- Remember to adjust the input mapping of your transformation(s) to use the tables extracted from your selected data sources if necessary. Select the step and click **Edit Configuration**.
+- Save the changes.
+
+Once you’ve built your flow end-to-end, it may look something like this:
+
+{: .image-popup}
+![Add SQL Transformation](/tutorial/automate/automate10.png)
+   
+Click **Run Flow** to start the data pipeline.
+
+## Execute Tasks in Parallel
+You can group multiple tasks within one step, a phase. These tasks then run independently in parallel, speeding up the execution. 
+Steps execute sequentially, while tasks within a single step run in parallel. If you have multiple extractors, you can include them all in a single step, allowing them to run simultaneously. 
+
+The same applies to data writers. Also, transformations independent of the connectors can be grouped within the same step. Note that this does not reduce costs, as each job consume credits independently.
+
+***Warning:** If too many tasks are scheduled in a single phase, you may exceed the available Storage job slots, causing delays in your flow’s execution. Limiting the number of concurrent component jobs 
+to 10 is recommended to avoid reaching Storage capacity limits. You can, of course, configure your flows to execute more jobs in parallel. Keboola will then concurrently execute the jobs to the maximum 
+extent possible based on available resources.*
+
+You can also set up parallelization **among components**, directly in the component's UI for [row-based components](/components/#configuration-rows) like database extractors using the same credentials to run multiple tables concurrently.
+
+{% include tip.html title="Execute Individual Configurations" content="
+Run configurations individually in Keboola Flows for more efficient workflow management. You can fine-tune which rows to run in advanced settings for greater 
+control.
+
+Use the menu next to each configuration row in the UI to execute specific configurations as needed, optimizing time and resources.
+" %}
+
+### Parallel limits
+Component jobs don't have a strict parallel limit, but [Storage jobs](/storage/jobs/) do. They are typically capped at 10 parallel jobs but the Keboola Support team can help you adjust this. 
+In environments with many users, component jobs may queue for Storage job availability, affecting runtime.
```

Comment:
Když už je tady o tom teda celej odstavec, tak by se to teda asi mělo vysvětlit líp, ale nevím jak moc jak teď z patra. Nicméně ten vztah mezi component jobs a storage jobs je všechno možný jen ne 1:1

Reply:
to bohuzel nevim, jak vyresit. Muzu nechat zmineny jen Storage jobs.

Reply:
jj, nechal bych tu jen tu zmínku a dal to tady asi radši nerozpatlával